src/risedevtool/src/task/postgres_service.rs

```diff
@@ -28,6 +28,13 @@ impl DockerServiceConfig for PostgresConfig {
         self.image.clone()
     }
 
+    fn args(&self) -> Vec<String> {
+        // Enable CDC.
+        ["-c", "wal_level=logical", "-c", "max_replication_slots=30"]
```

Comment:
According to:

https://github.com/risingwavelabs/risingwave/blob/3e881e699e7a7dd0934c98f474b40ca0111469af/ci/docker-compose.yml#L17

Reply:
Don't we need to include `postgres` in the command? 🤔

Reply:
It's `docker run <image> <command>`

Reply:
There seems to be some subtlety:

https://arc.net/l/quote/earthwwm

Anyway I've tested it locally.